<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDialogNewGame</class>
 <widget class="QDialog" name="QDialogNewGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>142</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>190</width>
    <height>142</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>190</width>
    <height>142</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Waree</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Players</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Waree</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cards</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Waree</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Size</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinPlayers">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>4</number>
   </property>
   <property name="value">
    <number>2</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinCards">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="minimum">
    <number>12</number>
   </property>
   <property name="maximum">
    <number>24</number>
   </property>
   <property name="singleStep">
    <number>12</number>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinSize">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimum">
    <number>5</number>
   </property>
   <property name="maximum">
    <number>11</number>
   </property>
   <property name="singleStep">
    <number>2</number>
   </property>
   <property name="value">
    <number>7</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>spinPlayers</tabstop>
  <tabstop>spinCards</tabstop>
  <tabstop>spinSize</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>QDialogNewGame</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>QDialogNewGame</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
